### PR TITLE
[DEV APPROVED] - Replace rubocop with mas-standards rubocop

### PR DESCRIPTION
**Pull request by tomas-stefano**
**Target Process ticket**

No ticket. Done on development learn time.

**Summary**

As I like challenges I decided to add our rubocop from [mas-standards](github.com/moneyadviceservice/mas-standards) repo into the frontend repo.

There was 1062 offences with the old rubocop and with the new a litle bit more. I fixed almost all of them but there are still almost 66 offences.

Another thing that I realised is that a lot of code not being used by the frontend. I really really like to delete them but this should be another work/PR on its own. Most of unused files lived under **lib/core** dir so I Ignored them on rubocop for now.

Also I decided to continue to the LineLength as 120 for now and I will return to 80 and fixed the offences in another PR as well in a next DLT.

**QA / Testing**

There should be some QA manual tests which I will do and ask a QA to do it as well.

**Checklist**

Make sure that when your PR is good to be reviewed you check the below checks:

+ [x] Tests passed.

### CODE REVIEW

File: app/controllers/concerns/chat.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ module Chat
 
   included do
     helper_method def chat_opening_hours
-                    @chat_opening_hours ||=
-                      ChatOpeningHoursDecorator.decorate(Rails.application.config.chat_opening_hours)
+      @chat_opening_hours ||=
+        ChatOpeningHoursDecorator.decorate(Rails.application.config.chat_opening_hours)
                   end
```
Comment (willhall88) [inline app/controllers/concerns/chat.rb]: this is an awkward indentation! I can see why the above lines were indented so much. if you move this `end` left as well, will it complain?

File: app/controllers/content_hub_controller.rb [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
 
-  def show
-  end
+  def show; end
 
   private
 
   def translation_prefix
-    "content_hub.#{params[:slug].gsub(/-/, '_')}"
+    "content_hub.#{params[:slug].tr('-', '_')}"
```
Comment (willhall88) [inline app/controllers/content_hub_controller.rb]: I learnt something new with this one!

File: app/controllers/cost_calculator_builder_controller.rb [hunk 3]
```diff
@@ -6,7 +6,7 @@ def category_id
   end
 
   # Note: syndicated_tool_request? is overridden in the engine.
-  #       This tool isn't served on the partner-tools subdomain,
+  #        This tool isn't served on the partner-tools subdomain,
```
Comment (willhall88) [inline app/controllers/cost_calculator_builder_controller.rb]: ❓ 🤔

File: app/helpers/budget_warning.rb [hunk 4]
```diff
@@ -14,7 +14,7 @@ def initialize
     end
 
     def go_live?
-      Time.zone.today >= date ? true : false
```
Comment (willhall88) [inline app/helpers/budget_warning.rb]: 🤦‍♂️

Comment (tomas-stefano) [reply app/helpers/budget_warning.rb]: haha same reaction from me.

File: app/controllers/concerns/chat.rb (hunk 1 above)
Comment (tomas-stefano) [reply app/controllers/concerns/chat.rb]: It was complaning actually:
```
app/controllers/concerns/chat.rb:8:19: W: Layout/DefEndAlignment: end at 8, 18 is not aligned with helper_method def at 5, 4.
                  end
```
I put the end on the right place and now we have 65 offences :D.

**Pull request merged**